CONTRIBUTING.md

```diff
@@ -17,26 +17,26 @@ Maven 3.0+ is required.
 
 When downloading the source, we recommend you obtain service account credentials. 
 These credentials will allow you to run integration tests using `mvn verify` in command line. 
-Follow step 2 of the [authentication instructions](https://github.com/GoogleCloudPlatform/gcloud-java#authentication) to generate and use JSON service account credentials.
+Follow step 2 of the [authentication instructions](https://github.com/GoogleCloudPlatform/google-cloud-java#authentication) to generate and use JSON service account credentials.
 
 It's also important to test that changes don't break compatibility with App/Compute Engine and when running elsewhere. 
-To run tests on different platforms, try deploying the apps available on the [gcloud-java-examples](https://github.com/GoogleCloudPlatform/gcloud-java-examples) repository. 
-End-to-end tests should ensure that gcloud-java works when running on the
+To run tests on different platforms, try deploying the apps available on the [google-cloud-examples](https://github.com/GoogleCloudPlatform/google-cloud-examples) repository.
+End-to-end tests should ensure that google-cloud works when running on the
```

Comment:
Not sure whether we should use the artifact or the repo name here. Probably the repo name.